crates/ox_content_ssg/src/ssg.css

```
@@ -1849,38 +1849,49 @@ a:hover {
-    width: 100%;
-    max-height: none;
-    transform: translateY(100%);
-    z-index: 99;
-    border-radius: 0;
+    top: calc(var(--octc-header-height) + 0.75rem);
+    left: max(var(--octc-mobile-gutter), env(safe-area-inset-left, 0px));
+    right: max(var(--octc-mobile-gutter), env(safe-area-inset-right, 0px));
+    bottom: calc(var(--octc-mobile-footer-height) + env(safe-area-inset-bottom, 0px) + 0.75rem);
+    width: auto;
+    max-height: calc(
+      100dvh - var(--octc-header-height) - var(--octc-mobile-footer-height) -
```

Stylelint (17.14.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)